pyproject.toml

```diff
@@ -43,6 +43,9 @@ ignore = [
 branch = true
 parallel = true
 source = ["pip_sync_faster", "tests/unit"]
+omit = [
+    "*/pip_sync_faster/__main__.py",
```

Comment:
This'll go into the cookiecutter, `__main__.py` will always be omitted from coverage